app/poros/movie.rb

```diff
@@ -1,9 +1,53 @@
 class Movie
-  attr_reader :title, :vote_average, :id
+  attr_reader :title, :vote_average, :id, :summary, :reviews
 
   def initialize(data)
     @title = data[:original_title]
     @id = data[:id]
     @vote_average = data[:vote_average].round(1)
+    @runtime = data[:runtime]
+    @genres = data[:genres]
+    @summary = data[:overview]
+    @cast = actor_details(data)
+    @reviews = review_details(data) 
+  end
+
+  def convert_runtime
+    hours = @runtime / 60
+    remaining_min = @runtime % 60 
+    "#{hours}:#{remaining_min}"
+  end
+
+  def all_genres
+    genres_names = @genres.map do |genre|
+      genre[:name]
+    end
+    genres_names.join(', ')
+  end
+
+  def actor_details(data)
```

Comment:
We can move this `#actor_details` into a `private` block in this poro class, I THINK it would be counted as encapsulation, as it is only able to be used within this class